javascript/javascript1/week3/itemRemoval.js

```diff
@@ -0,0 +1,18 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+  for (let i = 0; i < names.length; i++) {
+    if (names[i] == "Ahmad") {
+        names.splice(i, 1);
+    }
+}
```

Comment:
This is one way to do it.
If you want to find the index, you can also use the `indexOf` array method:
```js
const nameToRemoveIndex = names.indexOf(nameToRemove);
names.splice(nameToRemoveIndex, 1);
```